libcontainer/process_linux.go

```diff
@@ -255,6 +255,19 @@ func (p *initProcess) execSetns() error {
 	return nil
 }
 
+// parentError attempts to read any error from the parent so we have better context
+// about the cause of the failure. If no error is found then a new system error for
+// err and cause is returned.
+func parentError(pipe *os.File, err error, cause string) error {
+	if ierr := parseSync(pipe, func(sync *syncT) error { return nil }); ierr != nil {
```

Comment:
@cyphar the main one was not including nsenter, which was missing from the docs:
import _ "github.com/opencontainers/runc/libcontainer/nsenter"